docs/Installing.asciidoc

```diff
@@ -1126,7 +1126,7 @@ sudo ln --force --no-dereference --relative --symbolic /var/lib/pgsql/data.$newv
 [source,sh]
 ----
 sudo systemctl start postgresql
-sudo systemctl start openqa-webui openqa-scheduler openqa-livehandler openqa-gru
+sudo systemctl start openqa-webui openqa-scheduler openqa-livehandler openqa-gru openqa-websockets
```

Comment:
At least for starting this should be pulled in as dependency

Reply:
True. This would only leave `openqa-webui` and `openqa-scheduler` for starting. However, for stopping it is still required. I've just tested that.

Reply:
By the way, it would be nice if you would put the web socket server directly after `openqa-webui` (for stopping).

Reply:
Yesterday I followed the database upgrade guide on o3 and there was no need to stop or start websockets explicitly

Reply:
@okurz it is possible, when I upgraded osd from PostgreSQL 10 to 12 , all looked fine, but websockets service started to throws errors. And explicit restart fixed this.

I think there is better to be safe, than wait for result.

Reply:
Yes, I agree. Your approach is safer :)